src/pages/wallet/send.vue

```diff
@@ -271,6 +282,76 @@ export default {
       this.newTx.address = info.address;
       this.newTx.payment_id = info.payment_id;
     },
+    // helper for constructing a dialog for confirming transactions
+    confirmTransactionDialog(val) {
+      // the split_transfer rpc call can return several txs
+      const { feeList, amountList, destinations, metadataList, priority } = val.txData;
+      const totalFees = feeList.reduce((a, b) => a + b, 0) / 1e9;
+      const totalAmount = amountList.reduce((a, b) => a + b, 0) / 1e9;
+      // a tx can be split, but only sent to one address
+      const destination = destinations[0].address;
+      const note = this.newTx.note;
+      // not stored on chain, but we should only save a note to a completed tx
+      const isBlink = [0, 2, 3, 4, 5].includes(priority) ? true : false;
```

Comment:
```suggestion
      const isBlink = priority !== 1;
```

Reply:
this way of checking was based on what Doyle suggested

Reply:
Fair enough 👍